README.md

```diff
@@ -5,10 +5,13 @@
 
 ## Download Twitter Videos and Gifs directly to your android device. 
 
-### [Download the latest APK here](https://github.com/emmanuelkehinde/TwittaSave-Android/releases/download/v3.0/twittasave-release.apk)
+### [Download the latest APK here](https://github.com/emmanuelkehinde/TwittaSave-Android/releases/latest)
```

Comment:
Until I put in `latest` release, I might not be committing this particular fix. But I love the idea. Will consider it as soon as I get a release done.